doc/source/serve/production-guide/kubernetes.md

```diff
@@ -0,0 +1,182 @@
+```
+
+To deploy the example, we simply `kubectl apply` the CR.
+This creates the underlying Ray cluster, consisting of a head and worker node pod (see {ref}`_cluster-key-concepts-under-construction` for more details on Ray clusters), as well as the service that can be used to query our application:
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl get pods
+NAME                                                      READY   STATUS    RESTARTS   AGE
+ervice-sample-raycluster-qd2vl-worker-small-group-bxpp6   1/1     Running   0          24m
+rayservice-sample-raycluster-qd2vl-head-45hj4             1/1     Running   0          24m
+
+$ kubectl get services
+NAME                                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                          AGE
+kubernetes                                         ClusterIP   10.100.0.1       <none>        443/TCP                                          62d
+# Services used internally by the KubeRay controller.
+rayservice-sample-head-svc                         ClusterIP   10.100.34.24     <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+rayservice-sample-raycluster-qd2vl-dashboard-svc   ClusterIP   10.100.109.177   <none>        52365/TCP                                        24m
+rayservice-sample-raycluster-qd2vl-head-svc        ClusterIP   10.100.180.221   <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+# The Serve service that we will use to send queries to the application.
+rayservice-sample-serve-svc                        ClusterIP   10.100.39.92     <none>        8000/TCP                                         24m
+```
+
+## Querying the application
+
+Once the `RayService` is running, we can query it over HTTP using the service created by the KubeRay controller.
+This service can be queried directly from inside the cluster, but to access it from your laptop you'll need to configure a Kubernetes ingress or use port forwarding as below:
+
+```console
+$ kubectl port-forward service/rayservice-sample-serve-svc 8000
+$ curl -X POST -H 'Content-Type: application/json' localhost:8000 -d '["MANGO", 2]'
+6
+```
+
+## Getting the status of the application
+
+As the `RayService` is running, the `KubeRay` controller continually monitors it and writes relevant status updates to the CR.
+You can view the status of the application using `kubectl describe`.
+This includes the status of the cluster, events such as health check failures or restarts, and the application-level statuses reported by [`serve status`](serve-in-production-inspecting).
+
+```console
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl describe rayservice rayservice-sample
```

Comment:
Should we include an output for `describe` here?

Reply:
yeah we should, i'll add one

Reply:
finally tested all of the k8s commands and added this 👍